apcd_cms/src/client/src/hooks/admin/useAdmin.ts

```diff
@@ -76,7 +75,6 @@ const getSubmissionFilters = async () => {
 
 export const useSubmissionFilters = (): UseQueryResult<FilterOptions> => {
   const query = useQuery(['submissionFilters'], () => getSubmissionFilters(), {
-    staleTime: 5 * 60 * 1000,
     cacheTime: 10 * 60 * 1000,
     refetchOnWindowFocus: false,
   }) as UseQueryResult<FilterOptions>;
```

Comment:
I meant to remove all config settings, like this, so the defaults kick in.
```
const query = useQuery(['submissionFilters'], () => getSubmissionFilters()) as UseQueryResult<FilterOptions>;
```

Reply:
Changes applied!  [config settings removal](https://github.com/TACC/Core-CMS-Custom/pull/395/commits/3cef5502252ab9790f6a3a4d5bf1bed5e126dc56)